massa-cipher/src/tests/mod.rs

```diff
@@ -0,0 +1,40 @@
+#![cfg(test)]
+mod tests {
+    use crate::decrypt::decrypt;
+    use crate::encrypt::encrypt;
+
+    #[test]
+    fn test_encrypt() {
+        let password = "password";
+        let data = "data";
+
+        let result = encrypt(password, data.as_bytes());
+        assert!(result.is_ok());
+
+        let cipher_data = result.unwrap();
+        assert_eq!(cipher_data.encrypted_bytes.len(), 20);
+        assert_eq!(cipher_data.salt.len(), 16);
```

Comment:
constant: SALT_SIZE here?

Reply:
Wasn't aware of these values. Thank you. Fixed